app/services/notification_service.py

```
@@ -20,67 +24,246 @@
+
+        Use this from synchronous routers/endpoints.
+        """
+        try:
+            loop = asyncio.get_event_loop()
+        except RuntimeError:
+            # No event loop in current thread (sync context)
+            loop = asyncio.new_event_loop()
+            asyncio.set_event_loop(loop)
+
+        # Fire and forget - don't block router response
+        loop.create_task(self.dispatch_event(db, event_type, event_data))
```

Ruff (0.14.13)

Store a reference to the return value of `loop.create_task`

(RUF006)